packages/StepTracker/__tests__/StepTracker.spec.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { render, mount } from 'enzyme'
+
+import Icon from '@tds/core-decorative-icon'
+
+import StepTracker from '../StepTracker'
+
+describe('<StepTracker />', () => {
+  const defaultProps = {
+    current: 1,
+    steps: ['Login', 'Purchase', 'Checkout'],
+  }
+
+  const doMount = (overrides = {}) => {
+    const container = mount(<StepTracker {...defaultProps} {...overrides} />)
+    const innerSteps = container.find('[data-testid="innerStep"]')
+
+    return {
+      container,
+      innerSteps,
```

Comment:
Referencing my comment above about `data-testid`, if we use the step index when creating the testid, then this could become...

```js
return {
  container,
  findStep: (index) => container.find(`[data-testid="step-${index}"]`)
}
```

and then our tests would be...

```js
expect(findStep(0)).toContainReact(
  <Icon symbol="checkmark" size={16} variant="inverted" />
)
```